src/xrEngine/Stats.cpp

```diff
@@ -74,7 +74,8 @@ static optimizer vtune;
 //////////////////////////////////////////////////////////////////////
 // Construction/Destruction
 //////////////////////////////////////////////////////////////////////
-ENGINE_API bool g_bDisableRedText = false;
+ENGINE_API bool g_bDisableRedText = 0;
```

Comment:
Please, revert unneeded changes:
```suggestion
ENGINE_API bool g_bDisableRedText = false;
```